systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,22 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+<div class="box-container">
+  <h3>Organizers</h3>
+  <ul>
+   {% for organizer in object.organizers.all %}       
+      <li>
+        <a href="{{ organizer.get_absolute_url }}">{{ organizer }}</a> 
+      </li>
+    {% endfor %} 
+  </ul>
+  <h3>Members</h3>
```

Comment:
The members section should be paginated. As it is now we might get long lists of 100+ members. See in code, we have some pagination snippets for that.